v2/fb-xcom-openapi/fb-xcom-unified-openapi.yaml

```diff
@@ -487,6 +486,8 @@ paths:
           $ref: '#/components/responses/BadRequestResponse'
         '401':
           $ref: '#/components/responses/UnauthorizedResponse'
+        '404':
+          $ref: '#/components/responses/NotFoundResponse'
```

Comment:
I prefer not to specify explicitly the 404 response, because it means duplicating the same two lines for each endpoint. The error is quite straightforward imho. Just make sure the enum value is in `ErrorType`.